azplugins/bond.py

```diff
@@ -120,12 +120,12 @@ class fene(hoomd.md.bond._bond):
     - :math:`r_0` - size parameter ``r0`` (in distance units)
     - :math:`\varepsilon` - repulsive force strength ``epsilon`` (in energy units)
     - :math:`\sigma` - repulsive force interaction distance ``sigma`` (in distance units)
-
+    - :math:`\delta` - extra shift parameter for FENE bonds ``delta`` (in distance units)
```

Comment:
You will need to add an extra blank line after this bullet point to get the docs to render correctly. Can you also please document that the default value of delta is 0 in this bullet?